src/nni_manager/core/sqlDatabase.ts

```diff
@@ -203,7 +203,7 @@ class SqlDB implements Database {
     public storeMetricData(trialJobId: string, data: string): Promise<void> {
         const sql: string = 'insert into MetricData values (?,?,?,?,?,?)';
         const json: MetricDataRecord = JSON.parse(data);
-        const args: any[] = [Date.now(), json.trialJobId, json.parameterId, json.type, json.sequence, JSON.stringify(json.data)];
+        const args: any[] = [Date.now(), json.trialJobId, json.parameterId, json.type, json.sequence, json.data];
```

Comment:
maybe we should remove JSON.parse in this line https://github.com/microsoft/nni/blob/bcefce6a814c2c1228e25fcd98c88fea9e8111a9/src/nni_manager/core/nniDataStore.ts#L170?

Reply:
BTW, do we have unit test on data export and resume? maybe we should run IT for this pr

Reply:
Yes, we need data import/export IT test case.